src/transformers/models/patchtst/configuration_patchtst.py

```diff
@@ -0,0 +1,274 @@
+            The size of the target variable which by default is 1 for univariate targets. Would be > 1 in case of
+            multivariate targets.
+        context_length (`int`, *optional*, defaults to 32):
+            The context length for the encoder.
+        distribution_output (`str`, *optional*, defaults to `"student_t"`):
+            The distribution emission head for the model when loss is "nll". Could be either "student_t", "normal" or
+            "negative_binomial".
+        loss (`str`, *optional*, defaults to `"mse"`):
+            The loss function for the model corresponding to the `distribution_output` head. For parametric
+            distributions it is the negative log likelihood ("nll") and for point estimates it is the mean squared
+            error "mse".
+        patch_length (`int`, *optional*, defaults to 1):
+            Define the patch length of the patchification process.
+        patch_stride (`int`, *optional*, defaults to 1):
+            define the stride of the patchification process.
+        encoder_layers (`int`, *optional*, defaults to 3):
+            Number of encoder layers.
+        d_model (`int`, *optional*, defaults to 64):
+            Dimensionality of the transformer layers.
+        encoder_attention_heads (`int`, *optional*, defaults to 4):
+            Number of attention heads for each attention layer in the Transformer encoder.
+        shared_embedding (`bool`, *optional*, defaults to `True`):
+            Sharing the input embedding across all channels.
+        channel_attention (`bool`, *optional*, defaults to `False`):
+            Activate channel attention block in the Transformer to allow channels to attend each other.
+        encoder_ffn_dim (`int`, *optional*, defaults to 256):
+            Dimension of the "intermediate" (often named feed-forward) layer in encoder.
+        norm (`str` , *optional*, defaults to `"BatchNorm"`):
+            Normalization at each Transformer layer. Can be `"BatchNorm"` or `"LayerNorm"`.
+        norm_eps (`float`, *optional*, defaults to 1e-05):
+            A value added to the denominator for numerical stability of normalization.
+        attention_dropout (`float`, *optional*, defaults to 0.0):
+            The dropout probability for the attention probabilities.
+        dropout (`float`, *optional*, defaults to 0.0):
+            The dropout probability for all fully connected layers in the encoder, and decoder.
+        positional_dropout (`float`, *optional*, defaults to 0.0):
+            The dropout probability in the positional embedding layer.
+        dropout_path (`float`, *optional*, defaults to 0.0):
+            The dropout path in the residual block.
+        ff_dropout (`float`, *optional*, defaults to 0.0):
+            The dropout probability used between the two layers of the feed-forward networks.
+        bias (`bool`, *optional*, defaults to `True`):
+            Consider bias in the feed-forward networks.
```

Comment:
Maybe? Not sure if it's this 

```suggestion
            Whether to add bias in the feed-forward networks.
```